ilcsoft/mysql.py

```diff
@@ -46,4 +46,5 @@ def postCheckDeps(self):
             self.envpath["LD_LIBRARY_PATH"].append( "$MYSQL_HOME/lib" )
         self.envpath["LD_LIBRARY_PATH"].append( "$MYSQL_HOME/lib64/mysql" )
         self.envpath["LD_LIBRARY_PATH"].append( "$MYSQL_HOME/lib/mysql" )
+        self.addCMakeCache( "MySQL_DIR", self.installPath, "Path to MySQL directory" )
```

Comment:
@petricm This should fix the issue in https://github.com/iLCSoft/CondDBMySQL/pull/3/files#diff-ab792ce188708a5be46575410c93afeaR9 . After this PR is merged I'll update the TravisCI of CondDBMySQL.